docs/content/_navigation.json

```diff
@@ -1090,6 +1090,19 @@
           {
             "title": "Testing with Branch Deployments",
             "path": "/guides/dagster/branch_deployments"
+          },
+          {
+            "title": "Integrations",
```

Comment:
this is several layers deep. could we drop the Development? so it's just "Guides > Integrations"
<img width="327" alt="image" src="https://github.com/user-attachments/assets/329afc90-11ea-43aa-8e3a-8db8ac5a8e2b">

given we have the separate revamp effort, i'd just push for discoverability and ease of navigation (i.e. more flat IA).